### PR TITLE
Angular template

### PR DESCRIPTION
**Pull request by JamilOmar**

### CODE REVIEW

File: templates/angular-package/_env [hunk 1]
```diff
@@ -0,0 +1 @@
+TEST="test value with Rafa"
```
Comment (rafaelcalpena) [inline templates/angular-package/_env]: Does this file need to be commited?

File: templates/angular-package/config/default.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+    "test-config":"Hello from config"
+  
+}
```
Comment (rafaelcalpena) [inline templates/angular-package/config/default.json]: Does this file need to be commited?

File: templates/angular-package/_env (hunk 1 above)
Comment (JamilOmar) [reply templates/angular-package/_env]: hahahaha, thanks i will remove the value

File: templates/angular-package/config/default.json (hunk 2 above)
Comment (JamilOmar) [reply templates/angular-package/config/default.json]: I will add a default value, for the developers

**Pull request merged**